src/teleop/jetson/jetson_teleop.py

```diff
@@ -361,7 +368,7 @@ def main():
 
     ros.Subscriber("joystick", Joy, drive.teleop_drive_callback)
     ros.Subscriber("xbox/ra_control", Joy, arm.ra_control_callback)
-    ros.Subscriber("ra_slow_mode", Bool, arm.slow_mode_callback)
+    # ros.Subscriber("ra_slow_mode", Bool, arm.slow_mode_callback)
```

Comment:
why is this commented out? do we just delete it

Reply:
So now a button on the GUI just does nothing? that's kinda confusing

Reply:
Yeah I'll delete the stuff, forgot about this